### ISSUE TITLE
[Bug] Inventory Center not working correctly

### ISSUE THREAD

**Issue by tookender** [dicedpixels/hardcover#5]
![{9A915081-9DBE-4F28-9B7E-76CC8E7E674F}](https://github.com/user-attachments/assets/25e653b1-2061-40bd-8cee-b9c0d3c6ce9c)
The inventory center is not really centered like I expect it to be
I don't think the recipe book should be overlapping with the inventory?

**Comment by dicedpixels** [comment]
No, it shouldn't. The recipe book can go off screen but it shouldn't overlap. Can you send your latest.log? I feel like this could be a mod interaction I haven't encountered before.

**Comment by dicedpixels** [comment]
I've released 4.0.1+1.21.2 and 5.0.1+1.21.4 with a fix. I didn't have much time to test it, so let me know if the issue persist or if you notice anything weird.
